concordium-consensus/src/Concordium/External.hs

```diff
@@ -1398,3 +1410,4 @@ foreign export ccall checkIfRunning :: StablePtr ConsensusRunner -> IO Word8
 foreign export ccall freeCStr :: CString -> IO ()
 
 foreign export ccall importBlocks :: StablePtr ConsensusRunner -> CString -> Int64 -> IO Int64
+foreign export ccall stopImportingBlocks :: StablePtr ConsensusRunner -> IO ()
```

Comment:
This export must be added to the `lib.def` file.

Reply:
done 👍